### ISSUE TITLE
Reduce boilerplate: Move comment script out of this repo

### ISSUE THREAD

**Issue by stevector** [pantheon-systems/example-drops-8-composer#110]
Here is a break-out issue from #61 (comment)

The file [add-commit-comment](https://github.com/pantheon-systems/example-drops-8-composer/blob/7f15cf31478180494937651b7383e913e8439782/scripts/github/add-commit-comment) does not need to be specific to this repo. It could be moved to inside a Docker container after https://github.com/pantheon-systems/example-drops-8-composer/pull/107 lands or to a shared Composer project.

**Comment by stevector** [comment]
Ah, I think @greg-1-anderson intends to put this script inside the build tools plugin.

**Comment by greg-1-anderson** [comment]
Correct, Lukas is working on this. We can leave this issue open for tracking.
